drivers/entropy/Kconfig.wdt

```diff
@@ -0,0 +1,18 @@
+# Kconfig.wdt - race watchdog timer against system timer to generate entropy
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config ENTROPY_WDT_RNG
+	bool "Use watchdog timer jitter to generate entropy"
+	depends on WATCHDOG
```

Comment:
To recap verbal review comments. Let's have another config option that both WATCHDOG and ENTROPY_WDT_RNG can depend on. This way we can make sure the primary watchdog code is present, but also prevent the user from making a config that has both the WATCHDOG and the ENTROPY_WDT_RNG enabled.

Reply:
Also, do we need to have a way of indicating which watchdog devices have independent clocks? Do we know which of the existing drivers has this functionality?